internal/services/compute/virtual_machine_implicit_data_disk_from_source.go

```diff
@@ -0,0 +1,476 @@
+package compute
```

Comment:
This filename wants to be `virtual_machine_implicit_data_disk_from_source_resource.go`

Reply:
updated. Thanks.